databricks_template_schema.json

```diff
@@ -28,13 +28,13 @@
       "order": 5,
       "type": "string",
       "default": "",
```

Comment:
Does the `default` field support the `if-else` branch? Currently we check if the value is "" and set to the default value based on cloud.
We don't need the if-else in the description if we support `default` field.

Reply:
Good point. `if-else` is indeed supported by the default field. Changed to use it instead.